recipe/meta.yaml

```diff
@@ -1,37 +1,65 @@
-{% set version = "2.6.0" %}
-{% set sha256 = "7c425211a099f6fa6f74037e6e17be58fb5923b0bd11aea745e48ef83c488b49" %}
+{% set version = "3.0.1" %}
 
 package:
   name: singularity
   version: {{ version }}
 
 source:
-  fn: singularity-{{ version }}.tar.gz
-  url: https://github.com/sylabs/singularity/releases/download/{{ version }}/singularity-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  - folder: src/github.com/sylabs/singularity
+    git_url: https://github.com/sylabs/singularity.git
+    git_rev: v{{ version }}
+    patches:
+      - 0001-append-to-environment-flags.diff
+      - 0002-use-external-cni.diff
+      - 0003-fix-relocation.diff
 
 build:
   number: 1000
   skip: true  # [not linux]
 
 requirements:
   build:
+    - {{ compiler('cgo') }}
     - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
What is making use of C++?

Reply:
Their make system is looking for them during build-time. I will try a build without it and see if it works.